addon/data/set-installed-flag.js

```diff
@@ -4,9 +4,8 @@
  * http://mozilla.org/MPL/2.0/.
  */
 
-/* global cloneInto unsafeWindow */
+/* global unsafeWindow self */
 
 // Allow friendly clients to know that Test Pilot is installed.
-
 unsafeWindow.navigator.testpilotAddon = true;
```

Comment:
Ok, I finally understand it after staring at it for an hour. There are so many different contexts things run in!